src/lib/components/ui/Tabs/TabsItem.jsx

```diff
@@ -0,0 +1,85 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withProviderContext } from '../../../provider';
+import styles from './TabsItem.scss';
+
+export const TabsItem = ({
+  afterLabel,
+  beforeLabel,
+  id,
+  isActive,
+  label,
+  link,
+}) => (
+  <li
+    className={[
+      styles.root,
+      isActive ? styles.isRootActive : null,
+    ].join(' ')}
+    id={id}
+    key={link}
+  >
+    <a
+      href={link.href || link}
+      onClick={link.onClick}
+      className={styles.link}
+      id={id && `${id}__link`}
+    >
+      <span className={styles.linkContent}>
+        {beforeLabel}
+        <span
+          className={styles.label}
+          id={id && `${id}__label`}
+        >
+          {label}
+        </span>
+        {afterLabel}
+      </span>
+    </a>
+  </li>
+);
+
+TabsItem.defaultProps = {
+  afterLabel: null,
+  beforeLabel: null,
+  id: undefined,
+  isActive: false,
+};
+
+TabsItem.propTypes = {
+  /**
+   * Optional element shown after item's label, e.g. an icon.
+   */
+  afterLabel: PropTypes.node,
+  /**
+   * Optional element shown before item's label, e.g. an icon.
+   */
+  beforeLabel: PropTypes.node,
+  /**
+   * ID of the root HTML element. It also serves as a prefix for important inner elements:
+   * `<ID>__link`, `<ID>__label`.
+   */
+  id: PropTypes.string,
+  /**
+   * If `true`, item is marked as active.
+   */
+  isActive: PropTypes.bool,
+  /**
+   * Item's label.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Item's link. Can be a URI or an object with `href` and `onClick` function.
+   */
+  link: PropTypes.oneOfType([
```

Comment:
Is object definition necessary here? It would be more simple to have:

```js
link: PropTypes.oneOfType([
  PropTypes.string,
  PropTypes.func,
])
```

I do not see benefit in having mixed definition for a such simple purpose. I don't even know why to have `href` and `onClick` defined at the same time as `href` is never used. What more, this definition is not correct as it allows to have empty definition object resulting in exception even if prop type check is passed.